encord_agents/core/constants.py

```diff
@@ -0,0 +1,3 @@
+ENCORD_DOMAIN_REGEX = (
+    r"^https:\/\/(?:(?:cord-ai-development--[\w\d]+-[\w\d]+\.web.app)|(?:(?:dev|staging|app)\.encord\.com))$"
```

Comment:
Also need us suffix 🇺🇸 . I think optional us suffix on dev|staging|app but may need to check:

i.e: https://app.us.encord.com/ is the path to the base app

Reply:
@Jim-Encord fixed this now. Also added a unit test for the CORS regex just to make it clear what it should and shouldn't match.